capstone-frontend/src/components/Navbar.js

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import '../App.css';
+
+class Navbar extends Component {
+  constructor(props) {
+    super(props);
+    this.toggleNavbar = this.toggleNavbar.bind(this);
+    this.toggleNavbarLibrary = this.toggleNavbarLibrary.bind(this);
+    this.state = {
+      navCollapsed: true,
+      navLibraryCollapsed: true
+    };
+  }
+
+  toggleNavbar() {
+    this.setState({
+    navCollapsed: !this.state.navCollapsed,
+      navLibraryCollapsed: true
+    });
+  }
+
+  toggleNavbarLibrary() {
+    this.setState({
+      navLibraryCollapsed: !this.state.navLibraryCollapsed,
+    });
+  }
+
+  render() {
+    const navCollapsed = this.state.navCollapsed;
+    const navClassOne = navCollapsed ? 'collapse navbar-collapse' : 'collapse navbar-collapse show';
+    var navClassTwo = navCollapsed ? 'navbar-toggler navbar-toggler-right collapsed' : 'navbar-toggler navbar-toggler-right';
+    navClassTwo += " navbar-toggler navbar-toggler-right"
+
+    const navLibraryCollapsed = this.state.navLibraryCollapsed;
+    var showLib = navLibraryCollapsed ? '' : 'show';
+
+    const navLibClassOne = "nav-item dropdown d-sm-block d-md-none " + showLib;
+    const navLibClassTwo = "dropdown-menu " + showLib;
+
+    return (
+      <nav id="navbar" className="navbar navbar-expand-md navbar-dark bg-primary fixed-top text-center">
+        <button type="button" onClick={this.toggleNavbar} className={`${navClassTwo}`}>
```

Comment:
nit: Don't think the template literal for `className` is necessary since it's already a string